C/makefile

```diff
@@ -0,0 +1,14 @@
+IC=$(HOME)/instantclient_19_11
```

Comment:
The file should have a license / copyright header.

Reply:
Added Copyright header